src/components/Image/Image.tsx

```diff
@@ -17,37 +17,106 @@ export interface ImageProps extends TPAComponentProps {
   onLoad?: React.EventHandler<React.SyntheticEvent>;
   /** A callback to be called if error occurs while loading */
   onError?: React.EventHandler<React.SyntheticEvent>;
+  /** An experience to set while the image is fetched and loaded  */
+  loadingBehavior?: 'none' | 'blur';
 }
 
+interface Dimensions {
+  width: ImageProps['width'];
+  height: ImageProps['height'];
+}
```

Comment:
I tried to use `Pick` but the problem is that `width` & `height` are optional at the moment in `ImageProps`. Either way, the dimensions logic is probably going to change...